governance/reviews/template.md

```diff
@@ -146,14 +235,27 @@ Areas of potential future development include:
 *
 
 ### Governance Findings Table
-<!--- Add additional rows as necessary. For each finding described above, it should also be included here with further detail. --->
+<!--- Add additional rows as necessary. For each finding described above, it should also be included here with further detail. 
+
+Should be reported as Must fix:
+- Blocker: blocker
+
+Should be reported as "Areas for Improvement"
+- High: not blocker, but important
+- Medium/low: needs improvement
+
+--->
+
+| Finding Title | Importance | Description |
+|:------------- |:----------:|:------------|
+| *Title* | Blocker/High/Medium/Low | *detailed description* |
 
-| Finding Title | Importance | Description | Links | Notes & Impact |
-|:------------- |:----------:|:------------|:------|:---------------|
-| *Title* | Critical/High/Medium/Low | *detailed description* | *relevant links* | *additional notes and explanation of impact if appropriate* |
```

Comment:
Removed the "Links" and "Notes & Impact" column.
Those were causing duplication of content in title and description columns.

Reply:
So, if there are multiple links for a specific finding, how would that be handled?

Reply:
In the description?

In the Flatcar review, I've initially filled the "Links" column. 
Then I found I am adding same links in the description and removed the "Links" column.

Please take a closer look at the Findings Table columns at [WIP review doc for Flatcar](https://github.com/cncf/tag-contributor-strategy/blob/3da188814d42d1bb46e600ebd3b2b7ecb670a75a/governance/assessments/projects/flatcar/2023-08-15.md#governance-findings-table).

Reply:
I reverted this change now